app/models/concerns/relatable.rb

```diff
@@ -28,6 +28,8 @@ def self.get_related_model(counter_name)
       counter_name = counter_name.split(/ as /i)[-1]  # honor aliasing
       counter_name.gsub!('_count','')   # support cached count columns as well
 
+      counter_name.gsub!('qtls','qtl')  # unfortunate special case... :(
```

Comment:
@nowakowski Of course that was not the reason, it was in the next sentence ;P I wouldn't myself know upfront how that affects other layers. I think it could be added as a low priority issue, WDYT?